ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,19 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- floor_position_mode:
+  - BASEMENT
+  - UPPERGROUND
+  - LOWERGROUND
+  - GROUND
+  - FIRST
+  - SECOND
+  - THIRD
+  - FOURTH
+  - FOURTHMEZ
+  - PLANTROOM
+
+- max_load_alarm:
```

Comment:
This is the incorrect use of `max` if you are trying to model when the elevator is at its weight limit please use something like: `high_occupancy_alarm`

Reply:
changed thank you